ReactWindows/ReactNative/Animated/AnimatedNode.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+
+namespace ReactNative.Animated
+{
+    abstract class AnimatedNode
+    {
+        private List<AnimatedNode> _children;
+
+        public AnimatedNode(int tag)
+        {
+            Tag = tag;
+        }
+
+        public int Tag
+        {
+            get;
+        }
+
+        public IReadOnlyList<AnimatedNode> Children
+        {
+            get
+            {
+                return _children;
+            }
+        }
+
+        public long BFSColor
```

Comment:
We should get a stylecop GitHub plugin that makes snarky comments about .NET naming standards :-D

---
In reply to: [74442832](https://github.com/ReactWindows/react-native-windows/pull/583#discussion_r74442832) [](ancestors = 74442832)